src/components/SignIn/SignInForm/index.tsx

```diff
@@ -40,7 +38,7 @@ export const SignInForm = (props: SignInFormProps): JSX.Element | null => {
     <View testID={testIds.page.signin.signInForm}>
       <InputWithLabel
         autoCorrect={false}
-        autoComplete={strings.username}
+        autoComplete={strings.username as 'username'}
```

Comment:
Squeezing in a typescript error fix.

https://www.typescriptlang.org/docs/handbook/2/everyday-types.html#type-assertions.